core/src/main/java/com/klaytn/caver/transaction/type/FeeDelegatedSmartContractExecutionWithRatio.java

```diff
@@ -0,0 +1,367 @@
+        rlpTypeList.add(RlpString.create(Numeric.toBigInt(this.getGas())));
+        rlpTypeList.add(RlpString.create(Numeric.hexStringToByteArray(this.getTo())));
+        rlpTypeList.add(RlpString.create(Numeric.toBigInt(this.getValue())));
+        rlpTypeList.add(RlpString.create(Numeric.hexStringToByteArray(this.getFrom())));
+        rlpTypeList.add(RlpString.create(Numeric.hexStringToByteArray(this.getInput())));
+        rlpTypeList.add(RlpString.create(Numeric.toBigInt(this.getFeeRatio())));
+        rlpTypeList.add(new RlpList(senderSignatureRLPList));
+
+        byte[] encodedTransaction = RlpEncoder.encode(new RlpList(rlpTypeList));
+        byte[] type = new byte[] { (byte)TransactionType.TxTypeFeeDelegatedSmartContractExecutionWithRatio.getType() };
+        byte[] rawTx = BytesUtils.concat(type, encodedTransaction);
+
+        return Numeric.toHexString(Hash.sha3(rawTx));
+    }
+
+    /**
+     * Check equals txObj passed parameter and Current instance.
+     * @param txObj The AbstractFeeDelegatedTransaction Object to compare
+     * @param checkSig Check whether signatures field is equal.
+     * @return boolean
+     */
+    @Override
+    public boolean compareTxField(AbstractFeeDelegatedWithRatioTransaction txObj, boolean checkSig) {
+        if(!super.compareTxField(txObj, checkSig)) return false;
+        if(!(txObj instanceof FeeDelegatedSmartContractExecutionWithRatio)) return false;
+        FeeDelegatedSmartContractExecutionWithRatio feeDelegatedSmartContractExecutionWithRatio = (FeeDelegatedSmartContractExecutionWithRatio)txObj;
+
+        if(!this.getTo().toLowerCase().equals(feeDelegatedSmartContractExecutionWithRatio.getTo().toLowerCase())) return false;
+        if(!Numeric.toBigInt(this.getValue()).equals(Numeric.toBigInt(feeDelegatedSmartContractExecutionWithRatio.getValue()))) return false;
+        if(!this.getInput().equals(feeDelegatedSmartContractExecutionWithRatio.getInput())) return false;
+
+        return true;
+    }
+
+    /**
+     * Getter function for to
+     * @return String
+     */
+    public String getTo() {
+        return to;
+    }
+
+    /**
+     * Getter function for value
+     * @return String
+     */
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Getter function for input
+     * @return String
+     */
+    public String getInput() {
+        return input;
+    }
+
+    /**
+     * Setter function for to
+     * @param to The account address that will receive the transferred value.
+     */
+    public void setTo(String to) {
+        if(to == null) {
+            throw new IllegalArgumentException("to is missing.");
+        }
+
+        if(!Utils.isAddress(to)) {
+            throw new IllegalArgumentException("Invalid address. : " + to);
+        }
+
+        this.to = to;
+    }
+
+    /**
+     * Setter function for value
+     * @param value The amount of KLAY in peb to be transferred.
+     */
+    public void setValue(String value) {
```

Comment:
How about adding value setter for `BigInteger`?

Reply:
I added it. :)